catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/dashboards/DashboardResource.java

```diff
@@ -208,10 +216,16 @@ public Dashboard get(
               description = "Fields requested in the returned resource",
               schema = @Schema(type = "string", example = FIELDS))
           @QueryParam("fields")
-          String fieldsParam)
+          String fieldsParam,
```

Comment:
Tangential topic:

@sureshms @amiorin  Should we support `?fields=*` for GET `{resource}/{id}` requests. The response would have everything in `FIELD_LIST`

Reply:
@mithmatt I don't have an opinion on this. I'd like to ask you to open an issue and continue the conversation there. I'm closing for now.